01-node-tutorial/answers/writeWithPromisesThen.js

```diff
@@ -0,0 +1,23 @@
+const fs = require('fs');
+
+// Write to temp.txt
+fs.promises.writeFile('temp.txt', 'Line 1\n')
```

Comment:
You should be able to just import the writeFile and readFile functions from the fs.promises module. Check the lesson overview for more info!